### ISSUE TITLE
More deploy options?

### ISSUE THREAD

**Issue by derdritte** [octopress/deploy#48]
Hi there,

I've spent the last day updating my octopress blog to 3.0, realizing it's still a work in progress.
Now the issue is that I publish my blog on a server that will (sadly) not support either rsync nor git (not even ssh).

This all leads me to say that I need a deployment method that supports ftp and/or the possibility to just enter a command (I was thinking of lftp).
Is 'FTP' or 'Command' planned as an option?
Although my ruby is pretty much non-existent, I could probably hack it in and submit a pull-request. Are more options welcome or are you trying to keep the deploy low key? I also read the issue about dandelion which actually does do ftp, so this would solve it for me as well.

**Comment by imathis** [comment]
I've looked into this. FTP doesn't have default support for features like only syncing changed files, optional deletion of unmatched files, and setting permissions on upload. These features already exist in rsync and replicating them for FTP will take a considerable effort. I've looked for existing tools that offer this sort of feature, but most things I've found are either buggy, or old and abandoned projects.

If anyone can point me to a quality, maintained ruby gem which adds these features to FTP I'll consider adding support. I've looked at Git-FTP, but at this point it doesn't look stable enough for me to consider.
